Formula/singular.rb

```diff
@@ -0,0 +1,43 @@
+class Singular < Formula
+  desc "Computer algebra system for polynomial computations."
+  homepage "https://www.singular.uni-kl.de/"
+  url "http://www.mathematik.uni-kl.de/ftp/pub/Math/Singular/src/4-1-0/singular-4.1.0p3.tar.gz"
+  version "4.1.0"
+  sha256 "440164c850d5a1575fcbfe95ab884088d03c0449570d40f465611932ffd0bf80"
+  head "https://github.com/Singular/Sources.git"
+
+  option "without-test", "Skip build-time tests (not recommended)"
+  option "without-flint", "Build without flint support"
+
+  depends_on :xcode => :build
```

Comment:
are you sure this is necessary and that the CLT is not sufficient? try building without this line please